pages/instructorFileEditor/instructorFileEditor.ejs

```diff
@@ -1,78 +1,155 @@
-<!DOCTYPE html>
-<html class="h-100">
-
 <head>
     <%- include('../partials/head'); %>
     <script src="/javascripts/lodash.min.js"></script>
     <script src="/javascripts/d3.min.js"></script>
     <link href="/stylesheets/theme.bootstrap.css" rel="stylesheet" />
-    <script src="/javascripts/ace/ace.js"></script>
+    <script src="/node_modules/ace-builds/src-min-noconflict/ace.js"></script>
+    <script src="/node_modules/crypto-js/crypto-js.js"></script>
```

Comment:
Is this used on the page?

Reply:
If you are referring to `crypto-js`, then yes - it's used for hashing in the client. Perhaps something lighter-weight would do, this was convenient (used on both client and server side).

Reply:
Seems fine to me